### PR TITLE
Update to ensure Microsoft isn't forgotten

### PR DESCRIPTION
**Pull request by drtootsie**
I felt as though it was unfair that hubot only thought about Apple.  We must praise the Microsoft

### CODE REVIEW

**Comment by AnthonyRoss** [comment]
+1

**Comment by jvarness** [comment]
gd it Pepper....... +1

**Comment by krwillxyz** [comment]
+1

**Comment by alex-bezek** [comment]
+1

**Comment by peterhle** [comment]
:+1:

File: src/rules.coffee [hunk 1]
```diff
@@ -20,10 +20,19 @@ otherRules = [
...
+  "A developer must obey any orders given to it by Microsoft, except where such orders would conflict with the First Law.",
+  "A developer must protect its own existence as long as such protection does not conflict with the First or Second Law."
+  "A developer should praise the all knowing and all seeing Overlord of Microsoft, The Bill Gates"
+  ]
+
 module.exports = (robot) ->
   robot.respond /(what are )?the (three |3 )?(rules|laws)/i, (msg) ->
     text = msg.message.text
-    if text.match(/apple/i) or text.match(/dev/i)
+    if text.match(/micro/i) or text.match(/oop/i)
```
Comment (pcarn) [inline src/rules.coffee:33]: /oop/i - Are you talking about object oriented programming? ;)

Comment (drtootsie) [inline src/rules.coffee:33]: No, our bot who uses this specifically has the word oop and dev in it.  In an attempt to not break others who use this script, I made sure it print out correctly for our bot :)

Comment (pcarn) [inline src/rules.coffee:33]: Can you add your case after the apple one then, so current functionality is not affected?

Comment (drtootsie) [inline src/rules.coffee:33]: @pcarn If I did that, it'd ruin the main part of the pull request.  As I was saying earlier, the bot I use contains the phrase "dev" and "oop" in it, therefore, in order for me to circumvent it always saying "Praise the Apple Gods," I'm minimizing how much of the functionality is being changed.  It seems much smarter and actually truthful to say "Praise the Gods of Microsoft" or "Bill Gates is our Savior."  It just seems silly to have it be an Apple biased script :)

Comment (pcarn) [inline src/rules.coffee:33]: lol good luck then ;)

**Comment by jvarness** [comment]
@hubot-scripts +1?

**Comment by joeyguerra** [comment]
stale

**Pull request closed without merging**